README.md

```
@@ -66,7 +67,13 @@
 ```
 This will prompt you for the necessary information step by step.
 
-6. To display help information:
+6. To launch the GUI:
```

Markdownlint

Expected: 1; Actual: 6; Style: 1/1/1
Ordered list item prefix

(MD029, ol-prefix)